url8be3f02398_vnamboo_LearnJava_tgz-pJ8-org_json_TestJ8/src/com/scb/cleanCode/AcceptOnlyOnceFilter.java

```diff
@@ -0,0 +1,22 @@
+package com.scb.cleanCode;
+
+import java.io.File;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class AcceptOnlyOnceFilter extends AbstractFileFilter {
+
+	@Override
+	public boolean accept(File file) {
+		// TODO Auto-generated method stub
+		return false;
+	}
+	
+	public static void main(String[] args) throws ParseException {
+		SimpleDateFormat sdf=new SimpleDateFormat("dd-MMM-yyyy HH:mm:ss");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.